REVERSE_PROXY.md

```diff
@@ -0,0 +1,375 @@
+# FDO Management API Reverse Proxy Setup
+
+This document provides instructions for setting up nginx or Apache as a reverse proxy with basic authentication for FDO management APIs.
+
+## Overview
+
+The FDO server exposes management APIs on `/api/v1/` endpoints without built-in authentication. A reverse proxy can provide:
+
+- Basic HTTP authentication
+- TLS termination
+- Access control and security headers
+- Rate limiting (additional protection beyond built-in limits)
+
+## Management API Endpoints
+
+The following endpoints require protection:
+
+### Manufacturing Service (typically port 8038)
+- `GET /api/v1/rvinfo` - Retrieve rendezvous information
+- `POST /api/v1/rvinfo` - Create rendezvous information
+- `PUT /api/v1/rvinfo` - Update rendezvous information
+- `GET /api/v1/vouchers?guid={guid}` - Download ownership vouchers
+
+### Owner Service (typically port 8043)
+- `GET /api/v1/owner/redirect` - Retrieve owner redirect information
+- `POST /api/v1/owner/redirect` - Create owner redirect information
+- `PUT /api/v1/owner/redirect` - Update owner redirect information
+- `POST /api/v1/owner/vouchers` - Upload ownership vouchers
+- `GET /api/v1/to0/{guid}` - Trigger TO0 protocol
+- `GET /api/v1/owner/resell/{guid}` - Resell device
+
+## Common Setup Steps
+
+### 1. Install Required Packages
+
+```bash
+# For nginx
+sudo dnf install nginx httpd-tools
+sudo systemctl enable nginx
+
+# For Apache  
+sudo dnf install httpd mod_ssl httpd-tools
+sudo systemctl enable httpd
+```
+
+### 2. Create Password File
```

Comment:
I believe the passwords will be hashed after they are entered, so I think retrieving them after the fact would be difficult.  Maybe a note here about recording the password for later use would be smart.